GAMES.json

```diff
@@ -1183,12 +1184,17 @@
 	"323720": true,
 	"324070": true,
 	"324120": true,
+	"324260":
+	{
+		"Comment": "Requires Mono"
```

Comment:
This isn’t strictly something we cover in this db ‐ https://github.com/SteamDatabase/SteamLinux#contribution